apollo-router/src/spec/schema.rs

```diff
@@ -38,24 +42,353 @@ pub(crate) struct Schema {
     root_operations: HashMap<OperationKind, String>,
 }
 
+/// YAML-like representation with sorted hashmap/sets, more amenable to diffing
+impl std::fmt::Debug for Schema {
```

Comment:
An alternative could be to replace all `HashMap`/`HashSet`s with `IndexMap`/`IndexSet` so that `derive(Debug)` output would be stable, but this might impact runtime performance: https://github.com/bluss/indexmap#performance